ci/vars.yml

```diff
@@ -0,0 +1,6 @@
+emr-launcher-version:
+  development: "1.0.41"
+  qa:          "1.0.41"
+  integration: "1.0.41"
+  preprod:     "1.0.41"
+  production:  "1.0.41"
```

Comment:
Are these used?

Reply:
but the emr-launcher-version variable isn't used in any ci jobs as far as I can tell